dataactvalidator/filestreaming/csvAbstractReader.py

```diff
@@ -19,10 +19,12 @@ class CsvAbstractReader(object):
     def openFile(self,region,bucket,filename,csvSchema,bucketName,errorFilename):
         """ Opens file and prepares to read each record, mapping entries to specified column names
         Args:
-            bucket : the S3 Bucket
+            region: AWS region where the bucket is located
```

Comment:
It looks like those values are passed to getWriter, which only uses them if self.isLocal is false.  This function is overloaded by CsvLocalReader.openFile and CsvS3Reader.openFile, which therefore need to have the same arguments,  Those arguments are simply ignored if the object is a CsvLocalReader rather than a CsvS3Reader.  I'll add a note to these arguments that they are not used if running locally.